extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -278,6 +298,17 @@ export class AmpAnalytics extends AMP.BaseElement {
   }
 
   /**
+   * Receives any response that may be sent from the cross-domain iframe
+   * @param {!string} type The type parameter of the cross-domain iframe
+   * @param {Object} response The response message from the iframe
```

Comment:
Done, but to be honest I don't like it. The iframe creates an arbitrary object, which is passed to sendMessageToCreative(), which then wraps that in a AmpAnalytics3pResponse. I liked the symmetry of the amp-analytics tag receiving exactly what the iframe created, with Transport doing the unwrapping. Now that AmpAnalytics.processCrossDomainIframeResponse_() is receiving an AmpAnalytics3pResponse, that symmetry is broken. (We could restore it by requiring the iframe to create the AmpAnalytics3pResponse directly, but I am less in favor of that.)